app/perftools/management/commands/create_jtbd_cache.py

```diff
@@ -61,7 +61,7 @@ def create_jtbd_earn_cache():
 
     featured_grant = None
 
-    if not settings.DEBUG:
+    if 'prod' in settings.ENV:
```

Comment:
would setting.ENV be a string and not an array ?

Reply:
Yes! I swapped them by accident. Thank you for catching that!